fudge-ratio/src/main/webapp/script.js

```diff
@@ -0,0 +1,38 @@
+
+
+var seconds = 0;
```

Comment:
why are we using var? (here is a article on the pitfalls of var: https://levelup.gitconnected.com/stop-using-var-to-declare-variables-in-javascript-6c0caec16f43)